doc/api/next_api_changes/removals/21980-CC.rst

```diff
@@ -0,0 +1,4 @@
+Removed logging
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Removed `loaded modules` logging because it can produce extensive output.
```

Comment:
Maybe something more like "The list of currently loaded modules is no longer logged at the DEBUG level at Matplotlib import time.  If you were relying on this please arrange for your own logging." ?

Reply:
This is going to be read by people maybe a few years in the future so it needs to have enough context that it makes sense without tracking down the issue that this PR closed.

Reply:
I think the message is more helpful now.